app/src/main/java/com/kickstarter/services/KSUri.java

```diff
@@ -14,6 +14,10 @@
 public final class KSUri {
```

Comment:
We should migrate this to kotlin at some point.

Reply:
nw, I have another branch with the kotlin migration ;) it will be another separeted PR

Reply:
Awesome! 😃